packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/Abs_L1TokenGateway.sol

```diff
@@ -53,7 +53,7 @@ abstract contract Abs_L1TokenGateway is iOVM_L1TokenGateway, OVM_CrossDomainEnab
      ********************************/
 
     // Default gas value which can be overridden if more complex logic runs on L2.
-    uint32 public DEFAULT_FINALIZE_DEPOSIT_L2_GAS = 1200000;
+    uint32 internal constant DEFAULT_FINALIZE_DEPOSIT_L2_GAS = 1200000;
```

Comment:
@ben-chain is there any reason this value should remain public?
There's a getter below, so I can't imagine there is.

Reply:
I think this is the right move since there is a getter.